### PR TITLE
plan: tiny clean up

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

**Comment by coocood** [comment]
@zimulala 
The email is not properly set in your git config.

File: plan/task.go [hunk 1]
```diff
@@ -465,9 +465,6 @@ func (p *PhysicalHashAgg) newPartialAggregate() (partialAgg, finalAgg *PhysicalH
 	}
 	partialAgg = p.Copy().(*PhysicalHashAgg)
 	// TODO: It's toooooo ugly here. Refactor in the future !!
```
Comment (winoros) [inline plan/task.go]: Does this TODO still have meaning?

**Comment by shenli** [comment]
LGTM

File: plan/task.go (hunk 1 above)
Comment (zz-jason) [reply plan/task.go]: the todo message is too abstract here, how about making it more specific ?

**Pull request closed without merging**